docs/variants/protectli_vp66xx/post-install.md

```diff
@@ -0,0 +1,18 @@
+# Post-installation setup
+
+This document contains extra steps to perform after installing Dasharo in order
+to enable full functionality.
+
+## Windows
+
+This section covers Windows post-install steps tested on Windows 11. It is
+likely that similar procedures would for others Linux distributions as well.
```

Comment:
```suggestion
likely that similar procedures would <missing work here?> for others Linux distributions as well.
```

Reply:
Why it is likely for Linux distros, if it happens on Windows? There was any problem with Ubuntu?

Reply:
No, that was a leftout sentence from the vp46xx's post-install.